src/forms/TextInput.jsx

```diff
@@ -115,6 +115,7 @@ export class TextInput extends React.Component<Props> {
 						pattern={pattern}
 						disabled={disabled}
 						id={id}
+						aria-label={label ? undefined : id || name}
```

Comment:
It's a bit difficult to understand whether `|| name` is a fallback for the `id` or the entire ternary operator. Could you add round brackes to explicitly specify precedence of operations?

Reply:
@mkleymenov it was added, but prettier removed it :)